opencti-platform/opencti-front/src/private/Root.tsx

```diff
@@ -302,6 +305,11 @@ const RootComponent: FunctionComponent<RootComponentProps> = ({ queryRef }) => {
   const bannerSettings = computeBannerSettings(settings);
   const platformModuleHelpers = platformModuleHelper(settings);
   const platformAnalyticsConfiguration = generateAnalyticsConfig(settings);
+  const overviewLayoutCustomizationEntries = entitySettingsData
```

Comment:
I think you can make it simplier with new Map(Object.entries....)

Like :

```
const overviewNoMap = {
    'Threat-Actor-Individual': new Map(Object.entries(threatActorIndividualOverviewLayout)),
  };
  const overviewLayoutCustomization = new Map(Object.entries(overviewNoMap));

```

Reply:
Typing was not allowing this